### PR TITLE
chronam -> openoni

### PR DESCRIPTION
**Pull request by jduss4**
Uses "openoni" instead of "open-oni" in order to help
make django naming conventions more readable
Not all aspects have been tested, but batch loading, viewing of essential
pages, searching, and navigation are working.

Does not include any rebranding (for example updating the about / api pages)

### CODE REVIEW

**Comment by jduss4** [comment]
Apologies to @jechols if this is not the kind of thing you want to be looking over!

File: core/templates/about_api.html [hunk 1]
```diff
@@ -26,8 +26,8 @@
 different links take you to get started.  When describing Chronicling America as the source of content, please use the URL and a Web site citation, such as &quot;from the Library of Congress, Chronicling America: Historic American Newspapers site&quot;.</p> 
 
 <p>For more information about the open source Chronicling America software 
-please see the <a href="https://github.com/LibraryofCongress/chronam">LibraryOfCongress/chronam</a> GitHub site.
-Also, please consider subscribing to the <a href="http://listserv.loc.gov/archives/chronam-users.html">ChronAm-Users</a> discussion list
+please see the <a href="https://github.com/LibraryofCongress/openoni">LibraryOfCongress/openoni</a> GitHub site.
+Also, please consider subscribing to the <a href="http://listserv.loc.gov/archives/openoni-users.html">openoni-Users</a> discussion list
```
Comment (jechols) [inline core/templates/about_api.html]: These changes result in invalid URLs

File: install_redhat.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ When you install mysql-server, you will be prompted for a root password. If
...
-Get chronam
+Get openoni
 -----------
 
-    sudo mkdir /opt/chronam
-    sudo chown $USER:users /opt/chronam
-    git clone https://github.com/LibraryOfCongress/chronam.git /opt/chronam
+    sudo mkdir /opt/openoni
+    sudo chown $USER:users /opt/openoni
+    git clone https://github.com/LibraryOfCongress/openoni.git /opt/openoni
```
Comment (jechols) [inline install_redhat.md]: This should be our URL

File: install_redhat.md [hunk 3]
```diff
@@ -71,18 +71,18 @@ If not, install GraphicsMagick:
...
 
-    sudo cp /opt/chronam/conf/chronam-canonical.conf /etc/httpd/conf.d/
+    sudo cp /opt/openoni/conf/openoni-canonical.conf /etc/httpd/conf.d/
 
 
 Continue
 --------
 
-* You can now return to the Install section in [README.md](https://github.com/LibraryOfCongress/chronam/blob/master/README.md#install)
+* You can now return to the Install section in [README.md](https://github.com/LibraryOfCongress/openoni/blob/master/README.md#install)
```
Comment (jechols) [inline install_redhat.md]: This needs to be a link to our repo - there's a cool way to link from markdown files to other files in the same repository if you feel like figuring that out, but I don't recall it offhand.

File: install_ubuntu.md [hunk 4]
```diff
@@ -7,14 +7,14 @@ The following are instructions for installing system level dependencies on Ubunt
...
 
-Next you need to get the chronam code:
+Next you need to get the openoni code:
 
-    sudo mkdir /opt/chronam
-    sudo chown $USER:users /opt/chronam
-    git clone https://github.com/LibraryOfCongress/chronam.git /opt/chronam
+    sudo mkdir /opt/openoni
+    sudo chown $USER:users /opt/openoni
+    git clone https://github.com/LibraryOfCongress/openoni.git /opt/openoni
```
Comment (jechols) [inline install_ubuntu.md]: Need proper github link

File: install_ubuntu.md [hunk 5]
```diff
@@ -28,26 +28,26 @@ Download Solr from a mirror site
...
+    sudo a2ensite openoni
+    sudo install -o $USER -g users -d /opt/openoni/static
+    sudo install -o $USER -g users -d /opt/openoni/.python-eggs
     sudo service apache2 reload
 
 Continue
 --------
 
-* You can now return to the Install section in [README.md](https://github.com/LibraryOfCongress/chronam/blob/master/README.md#install)
+* You can now return to the Install section in [README.md](https://github.com/LibraryOfCongress/openoni/blob/master/README.md#install)
```
Comment (jechols) [inline install_ubuntu.md]: Same as above

File: scripts/extract_marc_values.py [hunk 6]
```diff
@@ -7,11 +7,11 @@
...
 
 To run this file: python extract_cntry_from_worldcatrecs.py $path_to_bib_folder $FIELD $SUBFIELD
-Example: python extract_marc_values.py /vol/ndnp/chronam/bib/holdings/ 852 a
+Example: python extract_marc_values.py /vol/ndnp/openoni/bib/holdings/ 852 a
 
 
-This also useds: Chronam
-Found here: http://github.com/LibraryOfCongress/chronam
+This also useds: openoni
+Found here: http://github.com/LibraryOfCongress/openoni
```
Comment (jechols) [inline scripts/extract_marc_values.py]: Same as above - change to our github link

**Comment by jechols** [comment]
There's no place to put this note on the code directly, so I apologize for the lack of context.

I think we need a second branch to convert URLs, but I'm not 100% certain exactly how that would be handled.  Maybe a new setting like BASE_APP_URI?

In some cases, like the flickr code and probably the models code, a hard-coded URL will make the app only work for one website... but I believe there are some cases where we do want to reference hard-coded URLs, such as for issue metadata lookup when loading batches.

This might need a search over each instance of "chronicling", and some time spent figuring each one out.  There are URLs but also some comments that should probably be fixed.

I think there's a bigger question of whether we even want to keep maintaining some of this code (e.g., I'm not sure the flickr stuff makes sense outside Chronicling America), but that's a discussion for another time I think.

**Comment by jduss4** [comment]
I agree, the more URL settings the better, so that we don't have to go through and hardcode things repeatedly.  Even with things like the metadata lookup, we might want to put the URLs in a settings file in case people want to override them?

There are a lot of comments and info on the about page, about api, etc, that need to be reviewed and changed to reflect the Open ONI identity rather than Chronicling America.  I believe that is flagged as a non-tech label, if anybody wants to pick it up.  :)

**Pull request merged**